pycode/memilio-epidata/README.rst

```diff
@@ -46,7 +46,7 @@ Dependencies
 
 Required python packages:
 
-- pandas>=1.2.2
+- pandas>=2.0.0
```

Comment:
Have you checked that no other libraries we use in your project require specific versions of pandas that are incompatible with the 2.+ versiosn?